hail/python/hail/docs/_templates/layout.html

```diff
@@ -146,36 +120,33 @@
         <div class="wy-side-nav-search">
           {% block sidebartitle %}
 
+          
+          {% include "searchbox.html" %}
+
+          {% endblock %}
+        </div>
+
+        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
           {% if logo and theme_logo_only %}
             <a href="{{ pathto(master_doc) }}">
           {% else %}
-            <a href="{{ pathto(master_doc) }}" class="icon icon-home"> {{ project }}
+            <a href="{{ pathto(master_doc) }}" class="icon icon-home"> {{ project }} Docs
```

Comment:
This moves the home link from above the search bar to below the search bar, which is fine. However, this moves grayscale antialiased text directly above sub pixel antialiased text and the subtle difference is jarring.

I'm pretty sure icon fonts aren't intended to be used this way anyway. Let's change the HTML to:
```html
<a href="{{ pathto(master_doc) }}"><span class="icon icon-home"></span>{{project}} Docs  ...
```

That confines the bad antialiasing to the icon, on which it's not noticeable and seemingly unavoidable, thanks to the RTD theme.

Reply:
I've made the change. Looking into it, it appears there should be little difference between the two: all icon-specific css is applied in the pseudo-element selector, and unfortunately webkit-font-smoothing is applied to the link, not just the FontAwesome font. Let me know if I missed something, curious.

<img width="1389" alt="Screenshot 2019-08-19 12 15 17" src="https://user-images.githubusercontent.com/5543229/63281601-11635800-c27b-11e9-868a-7d9e8c44e6e9.png">